src/client/src/app/app-routing.module.ts

```diff
@@ -16,7 +17,11 @@ const routes: Routes = [
   {path: 'create-message', component:ChatComponent},
   {path:'contact-page', component:ContactPageComponent},
   {path: 'create-merch', component: CreateMerchComponent},
-  {path: 'merch', component: MerchComponent},
+
+  {path: 'create-merch', component: CreateMerchComponent,
+  canActivate: [RolesGuard], data:{roles:["ADMIN"]}},
+
+
```

Comment:
I dont think we should adjust the style too much by adding all these empty lines